main/FatesHistoryInterfaceMod.F90

```diff
@@ -8787,6 +8578,18 @@ subroutine define_history_vars(this, initialize_variables)
                hlms='CLM:ALM', upfreq=group_hifr_complx, ivar=ivar, initialize=initialize_variables, &
                index = ih_gpp_si_age)
 
+          call this%set_history_var(vname='FATES_NPP_LU', units='kg m-2 s-1',        &
+               long='net primary productivity by age bin in kg carbon per m2 per second', &
+               use_default='inactive', avgflag='A', vtype=site_landuse_r8,               &
+               hlms='CLM:ALM', upfreq=group_hifr_complx, ivar=ivar, initialize=initialize_variables, &
+               index = ih_npp_si_landuse)
+
+          call this%set_history_var(vname='FATES_GPP_LU', units='kg m-2 s-1',        &
+               long='gross primary productivity by age bin in kg carbon per m2 per second', &
```

Comment:
Why the use of "age bin" here instead of land use?

Reply:
Good catch @glemieux; cut-and-paste typo. will change.